### ISSUE TITLE
Enrich members info returned by `/api/v1/cluster/members`

### ISSUE THREAD

**Issue by fmassot** [quickwit-oss/quickwit#1495]
Currently, the endpoint returns the list of members, each member with its a few info:

```json

[
  {
    "id": "node-muddy-Sp3u/1652963532",
    "listen_address": "127.0.0.1:7280",
    "is_self": true,
    "generation": 1652963532
  }
]
```

It would be nice to have the service and other relevant info. We could even return the node state?

**Comment by fulmicoton** [comment]
I say let's dump the whole chitchat node state. It's always nice for debugging / explaining how stuff works.

**Comment by fmassot** [comment]
Closed by #1708
